jetty-server/src/test/java/org/eclipse/jetty/server/HttpConnectionTest.java

```diff
@@ -1361,6 +1367,68 @@ public void testCONNECT() throws Exception
         }
     }
 
+    @Test
+    public void testBytesIn() throws Exception
+    {
+        String chunk1 = "0123456789ABCDEF";
+        String chunk2 = IntStream.range(0, 64).mapToObj(i -> chunk1).collect(Collectors.joining());
+        long dataLength = chunk1.length() + chunk2.length();
+        server.stop();
+        server.setHandler(new AbstractHandler()
+        {
+            @Override
+            public void handle(String target, Request jettyRequest, HttpServletRequest request, HttpServletResponse response) throws IOException
+            {
+                jettyRequest.setHandled(true);
+                IO.copy(request.getInputStream(), IO.getNullStream());
+
+                HttpConnection connection = HttpConnection.getCurrentConnection();
+                long bytesIn = connection.getBytesIn();
+                assertThat(bytesIn, greaterThan(dataLength));
```

Comment:
Can't we do better than this?   We should know exactly how many bytes have been sent on a connection.
At the very least we need to check it is greater than `dataLength + headerBytes`

Reply:
@gregw I don't want to count exactly bytes for headers, as they may depend on dates, version strings that have additional digits (e.g. 10.0.9 -> 10.0.10), etc.
I just want to make sure that "resumed" reads count all the content -- I verified that the test failed before the fix.

Reply:
hmmmm